ui/src/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,61 @@
+import { render, screen } from '@testing-library/react';
+import * as Components from '@components';
+import * as OutlineIcons from '@heroicons/react/outline';
+import '@testing-library/jest-dom';
+
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const button = (
+    <Components.Button
+        className="children:border-0"
+        disabled={false}
+        icon={<OutlineIcons.ArrowRightIcon className="h-4 w-4 text-teal-600" />}
+        iconPosition="RIGHT"
+        onClick={() => {
+            return true;
+        }}
+        title={'Click me'}
+    />
+);
+
+describe('Button tests', () => {
+    it('Renders button component without crashing', () => {
+        render(button);
```

Comment:
Not really a big issue, but you could call the render() method in a beforeEach(), instead of writting "render(button)" in each test, since the props for that button are not changing.